external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/package.scala

```diff
@@ -33,8 +33,8 @@ package object kafka010 {   // scalastyle:ignore
       .createWithDefaultString("10m")
 
   private[kafka010] val CONSUMER_CACHE_CAPACITY =
-    ConfigBuilder("spark.sql.kafkaConsumerCache.capacity")
-      .doc("The size of consumers cached.")
+    ConfigBuilder("spark.kafka.consumer.cache.capacity")
+      .doc("The number of consumers cached.")
```

Comment:
maximum number?

Reply:
Fixed and additionally added a note about the parameter soft limit nature.